aiohttp_devtools/runserver/serve.py

```diff
@@ -109,6 +109,9 @@ def serve_main_app(config: Config, tty_path: Optional[str], loop: asyncio.Abstra
     with set_tty(tty_path):
         setup_logging(config.verbose)
 
+        # imports the factory. This gives users a chance to register alternative event loops
+        config.app_factory
```

Comment:
It's not your fault, but this looks dumb and isn't clear.

If we decide to merge this I think it would be better to remove the `app_factory` property and rename `_import_app_factory > import_app_factory` and use that instead.

Reply:
I agree that making that function public would be a better option. Want me to refactor the function?

Reply:
Also, regarding my problem, namely being able to set my own event loop, and not use the standard one - details are in the issue #156 

There are no other details that I can provide - basically I don't know of a nice way to make the `runserver` command use the event loop that I want.

If you can suggest a better way, that' fine. I haven't investigated that much, so I don't have an overview of the project's structure.

Until then, the problem remains valid, and both the traceback of the error, and the code that generated it can be found in the issue.

Thanks for your time! :)